src/main/java/com/google/cloud/tools/appengine/cloudsdk/CloudSdk.java

```diff
@@ -651,6 +651,14 @@ public Builder inheritProcessOutput(boolean inheritProcessOutput) {
     }
 
     /**
+     * Sets the desired Java SDK path, used in devappserver runs and App Engine standard staging.
+     */
+    public Builder javaSdkPath(Path javaSdkPath) {
```

Comment:
@joaoandremartins let me know once you've addressed Brian's other issues and I'll approve.